build-tools/helm/spark-kubernetes-operator/values.yaml

```diff
@@ -0,0 +1,195 @@
+################################################################################
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+image:
+  repository: spark-kubernetes-operator
+  pullPolicy: IfNotPresent
+  tag: 0.1.0
+  # If image digest is set then it takes precedence and the image tag will be ignored
+  # digest: ""
+
+imagePullSecrets: [ ]
+
+operatorDeployment:
+  # Replicas must be 1 unless leader election is enabled
+  replicas: 1
+  # Strategy type must be 'Recreate' unless leader election is enabled
+  strategy:
+    type: Recreate
+  operatorPod:
+    priorityClassName: null
+    annotations: { }
+    labels: { }
+    affinity: { }
+    nodeSelector: { }
+    # Node tolerations for operator pod assignment
+    # https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/
+    tolerations: [ ]
+    # Topology spread constrains
+    # https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/
+    topologySpreadConstraints: [ ]
+    operatorContainer:
+      jvmArgs: "-XX:+UseG1GC -Xms3G -Xmx3G -Dfile.encoding=UTF8"
+      env:
+      envFrom:
+      volumeMounts: { }
+      resources:
+        limits:
+          cpu: "1"
+          ephemeral-storage: 2Gi
+          memory: 2Gi
+        requests:
+          cpu: "1"
+          ephemeral-storage: 2Gi
+          memory: 2Gi
+      probes:
+        port: 19091
+        livenessProbe:
+          periodSeconds: 10
+          initialDelaySeconds: 30
```

Comment:
Is there any reason for this `30s`? Could you add a comment why the default value of `livenessProbe` is insufficient? Or, need to be overridden?